javascript/config.yaml

```diff
@@ -27,9 +27,9 @@ language:
         - npm install
       command: pm2-runtime start app.js -i $(nproc)
     deno:
-      command: deno run --allow-net --allow-read --allow-env app.ts
+      command: pm2 start app.ts --interpreter="deno" --interpreter-args="run --allow-net"
     bun:
-      command: bun run app.ts
+      command: pm2-runtime start "NODE_ENV=production bun app.ts" -i max
```

Comment:
```suggestion
   deno:
      deps:
        - npm
      bootstrap:
        - npm install -g pm2
      command: pm2-runtime start "NODE_ENV=production deno run --allow-net --allow-read --allow-env app.ts" -i max
    bun:
      deps:
        - npm
      bootstrap:
        - npm install -g pm2
      command: pm2-runtime start "NODE_ENV=production bun app.ts" -i max
```

Reply:
> `pm2` is not installed in containers.


Change it to `pm2-runtime`.


> You will need to add
{{#deps}}
  RUN apk add {{{.}}}
{{/deps}}

> {{#bootstrap}}
  RUN {{{.}}}
{{/bootstrap}}

> after `WORKDIR` in `deno.Dockerfile`


Done.


> and bun install prior to CMD in bun.Dockerfile

Done.